### PR TITLE
ACQ - adding comments to Metadb po_lines_tags

### PR DESCRIPTION
**Pull request by sbeltaine**
Nassib: invoice_lines_adjustments commit should be removed

```
All queries:
- [ ] Pull request is based on a new branch (not main)
- [ ] Pull request scope is not overly broad
- [ ] Release notes have been added or updated in CHANGES.md
- [ ] Query runs without errors
- [ ] Query output is correct
- [ ] Query logic is clear and well documented
- [ ] Query is readable and properly indented with spaces (not tabs)
- [ ] Table and column names are in all-lowercase
- [ ] Quotation marks are used only where necessary

Report queries:
- [ ] Query has complete user documentation
    - [ ] Purpose of report
    - [ ] Sample output
    - [ ] Query instructions

Derived tables:
- [ ] Query begins with user documentation in comment lines
- [ ] File name is listed in `runlist.txt` after dependencies
- [ ] All columns have indexes
- [ ] Table is vacuumed and analyzed
```

### CODE REVIEW

**Comment by sbeltaine** [comment]
Addresses Issue #710. No changes to CHANGES.md because just adding comments. Updated runlist.txt with name of this file.

File: sql_metadb/derived_tables/invoice_lines_adjustments.sql [hunk 1]
```diff
@@ -52,4 +52,18 @@ CREATE INDEX ON invoice_lines_adjustments (adjustment_value);
...
+
+COMMENT ON COLUMN invoice_lines_adjustments.adjustment_prorate IS 'description';
+
+COMMENT ON COLUMN invoice_lines_adjustments.adjustment_relation_to_total IS 'description';
+
+COMMENT ON COLUMN invoice_lines_adjustments.adjustment_type IS 'description';
+
+COMMENT ON COLUMN invoice_lines_adjustments.adjustment_value IS 'description';
+
+COMMENT ON COLUMN invoice_lines_adjustments.adjustment_adjustments_total IS 'description';
```
Comment (stdombek) [inline sql_metadb/derived_tables/invoice_lines_adjustments.sql:67]: I think it was the accidentally uploaded file and shouldn't be part of this PR. Please remove when merging. @nassibnassar

**Review by stdombek — APPROVED** [review]
Everything looks good. Only comments were added to an existing table and an entry was made in the runlist.txt. Therefore no checklist.

@nassibnassar Please note that the one file `invoice_lines_adjustments.sql` should not be merged.

**Review by AlaoSUL — APPROVED** [review]
Comments for po_lines_tags.sql look good! Added to runlist.txt as well

As @stdombek mentioned above, please dont merge invoice_lines_adjustments.sql as it was accidently uploaded.

**Review by sbeltaine — COMMENTED** [review]
Viewed

**Review by sbeltaine — COMMENTED** [review]
Viewed

File: sql_metadb/derived_tables/runlist.txt [hunk 2]
```diff
@@ -54,6 +54,7 @@ po_lines_eresource.sql
 po_lines_location.sql
 po_lines_phys_mat_type.sql
 po_lines_physical.sql
+po_lines_tags.sql
```
Comment (nassibnassar) [inline sql_metadb/derived_tables/runlist.txt:57]: @sbeltaine Be careful, this was already in the runlist.

**Comment by nassibnassar** [comment]
merged in 5a34cfe
